src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const SHORT__TERM = 3;
+  const SHORT__DISCOUNT = 20;
+  const LONG__TERM = 7;
+  const LONG__DISCOUNT = 50;
```

Comment:
The constant names contain double underscores, which is unconventional and might be a typo. Usually, a single underscore is used in constant names to separate words. Consider renaming `SHORT__TERM` to `SHORT_TERM`, `SHORT__DISCOUNT` to `SHORT_DISCOUNT`, `LONG__TERM` to `LONG_TERM`, and `LONG__DISCOUNT` to `LONG_DISCOUNT` for better readability.